applications/ShapeOptimizationApplication/python_scripts/analyzer_internal.py

```diff
@@ -26,10 +28,32 @@
     from KratosMultiphysics.ConvectionDiffusionApplication.response_functions import convection_diffusion_response_function_factory as convdiff_response_factory
 except ImportError:
     convdiff_response_factory = None
-
+try:
+    from KratosMultiphysics.RANSApplication.response_functions import rans_response_function_factory as rans_response_factory
+except ImportError:
+    rans_response_factory = None
 
 import time as timer
 
+class IterationScope:

```

Comment:
Same concerns here w.r.t. the external solver as in #8388

Reply:
done :)